{"primaryContentSections":[{"declarations":[{"languages":["swift"],"platforms":["iOS"],"tokens":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"text":"invalidState","kind":"identifier"}]}],"kind":"declarations"}],"sections":[],"hierarchy":{"paths":[["doc:\/\/com.avaya.axp.client.sdk.AXPCore\/documentation\/AXPCore","doc:\/\/com.avaya.axp.client.sdk.AXPCore\/documentation\/AXPCore\/AXPError"]]},"kind":"symbol","abstract":[{"type":"text","text":"If the operation is not valid for the current state"}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/axpcore\/axperror\/invalidstate"]}],"identifier":{"interfaceLanguage":"swift","url":"doc:\/\/com.avaya.axp.client.sdk.AXPCore\/documentation\/AXPCore\/AXPError\/invalidState"},"metadata":{"modules":[{"name":"AXPCore"}],"title":"AXPError.invalidState","roleHeading":"Case","fragments":[{"text":"case","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"invalidState"}],"externalID":"s:7AXPCore8AXPErrorO12invalidStateyA2CmF","symbolKind":"case","role":"symbol"},"schemaVersion":{"patch":0,"major":0,"minor":3},"references":{"doc://com.avaya.axp.client.sdk.AXPCore/documentation/AXPCore/AXPError/invalidState":{"identifier":"doc:\/\/com.avaya.axp.client.sdk.AXPCore\/documentation\/AXPCore\/AXPError\/invalidState","abstract":[{"text":"If the operation is not valid for the current state","type":"text"}],"url":"\/documentation\/axpcore\/axperror\/invalidstate","fragments":[{"text":"case","kind":"keyword"},{"kind":"text","text":" "},{"text":"invalidState","kind":"identifier"}],"type":"topic","kind":"symbol","role":"symbol","title":"AXPError.invalidState"},"doc://com.avaya.axp.client.sdk.AXPCore/documentation/AXPCore":{"url":"\/documentation\/axpcore","title":"AXPCore","role":"collection","kind":"symbol","type":"topic","abstract":[{"type":"text","text":"The base module that establishes the session with Avaya Experience Platformâ„¢ (AXP) for the end user."}],"identifier":"doc:\/\/com.avaya.axp.client.sdk.AXPCore\/documentation\/AXPCore"},"doc://com.avaya.axp.client.sdk.AXPCore/documentation/AXPCore/AXPError":{"type":"topic","fragments":[{"text":"enum","kind":"keyword"},{"kind":"text","text":" "},{"text":"AXPError","kind":"identifier"}],"identifier":"doc:\/\/com.avaya.axp.client.sdk.AXPCore\/documentation\/AXPCore\/AXPError","navigatorTitle":[{"text":"AXPError","kind":"identifier"}],"url":"\/documentation\/axpcore\/axperror","title":"AXPError","abstract":[{"type":"text","text":"Error for the case where an operation failed due to an error contacting AXP"},{"type":"text","text":" "},{"type":"text","text":"across the network."}],"role":"symbol","kind":"symbol"}}}